src/Azure.Functions.Cli/Extensions/StringExtensions.cs

```diff
@@ -41,5 +43,22 @@ public static string SanitizeImageName(this string imageName)
 
             return cleanImageName.ToLowerInvariant().Substring(0, Math.Min(cleanImageName.Length, 128)).Trim();
         }
+
+        public static string ComputeSha256Hash(string rawData)
```

Comment:
nit: Since this is a class intended to create extension methods, it's better to be consistent and make this method also an extension method:

```suggestion
        public static string ComputeSha256Hash(this string rawData)
```